src/main/java/build/buildfarm/common/redis/RedisClient.java

```diff
@@ -109,8 +155,57 @@ public <T> T blockingCall(JedisInterruptibleContext<T> withJedis)
     return result;
   }
 
-  @SuppressWarnings("ConstantConditions")
   public <T> T call(JedisContext<T> withJedis) throws IOException {
+    return callImpl(withJedis);
```

Comment:
Minor: do we need to make a separate method for this still?

Reply:
fixed.  callImpl folded into call